spec/models/request_spec.rb

```diff
@@ -28,19 +29,21 @@
       end
 
       it 'has active_stage pointing to the first stage' do
-        expect(subject.as_json).to include('active_stage' => 3, 'total_stages' => 3)
+        expect(subject.as_json).to include('active_stage_number' => 3, 'total_stages' => 3)
```

Comment:
previous line has a typo, should be `has active_stage pointing to the last stage`